CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# 2.14.0 (2020-05-07)
```

Comment:
Please change the version to `# UNRELEASED`.  Our team will update this when we release a new version of `python-cloudant`.

Reply:
Done the changes